.gitignore

```diff
@@ -13,6 +13,7 @@ spec/reports
 test/tmp
 test/version_tmp
 tmp
+*~
```

Comment:
Better to put this in [your global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)